tools/star_fusion_1.10.1_gen_reference.cwl

```diff
@@ -40,7 +40,7 @@ inputs:
   inputBinding: { position: 3, prefix: "--fusion_annot_lib"} }
   annot_filter_rule: {type: File, doc: "target AnnotFilterRule.pm, from https://data.broadinstitute.org/Trinity/CTAT_RESOURCE_LIB",
   inputBinding: { position: 3, prefix: "--annot_filter_rule"} }
-  pfam_db: { type: 'string?', doc: "pfam database. If given will simply pull from current, and run hmmpress", default: 'current'
+  pfam_db: { type: 'string?', doc: "pfam database. If given will simply pull from current, and run hmmpress", default: 'current',
   inputBinding: { position: 3, prefix: '--pfam_db' } }
   dfam_db: { type: 'File[]',
```

Comment:
this linebreak approach is...confusing to read. i feel like there should be indenting.

Reply:
Yeah, I agree. Maybe should look into a linter for yaml